apps/studio/components/interfaces/QueryPerformance/QueryPerformanceGrid.tsx

```diff
@@ -54,24 +53,66 @@ export const QueryPerformanceGrid = ({ queryPerformanceQuery }: QueryPerformance
     (preset as QUERY_PERFORMANCE_REPORT_TYPES) ?? QUERY_PERFORMANCE_REPORT_TYPES.MOST_TIME_CONSUMING
 
   const columns = QUERY_PERFORMANCE_REPORTS[reportType].map((col) => {
+    const nonSortableColumns = ['query']
+
     const result: Column<any> = {
       key: col.id,
       name: col.name,
       resizable: true,
       minWidth: col.minWidth ?? 120,
+      sortable: !nonSortableColumns.includes(col.id),
```

Comment:
if we're introducing sorting - reckon we should probably also save the sort order in local storage, otherwise the sort order will keep getting reset on each tab session

Reply:
btw this might not be working - im trying on the staging preview but im not able to sort the columns by dragging them